sunpy/net/vso/tests/test_vso.py

```diff
@@ -327,54 +326,29 @@ def test_non_str_instrument():
         core_attrs.Instrument(1234)
 
 
-@pytest.mark.parametrize("waverange, as_dict", [
```

Comment:
Why has this test gone?

Reply:
The function it was testing wasn't being used anywhere so I removed it.